src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,82 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String supply_operation = "supply";
+    private static final String buy_operation = "buy";
+    private static final String result_header = "result";
+    private static final String comma = ",";
+    private static final int operation_type_ = 0;
+    private static final int amount_index = 1;
```

Comment:
CONSTANT_CASE uses uppercase
```suggestion
    private static final String SUPPLY_OPERATION = "supply";
    private static final String BUY_OPERATION = "buy";
    private static final String RESULT_HEADER = "result";
    private static final String SEPARATOR = ",";
    private static final int OPERATION_TYPE_INDEX = 0;
    private static final int AMOUNT_INDEX = 1;
```